lens-common/lens/private/compound/zoom.rkt

```diff
@@ -28,23 +28,151 @@ module+ test
      (make-isomorphism-lens
       (lens-transform zoom-lens _ transformer) ; (Outer A) -> (Outer B)
       (lens-transform zoom-lens _ inverse))]   ; (Outer B) -> (Outer A)
-    [transformer-lens
+    [trans-lens
```

Comment:
The shorter name is better, but I think we shouldn't bind this variable (using `_`) at all and should just rename the argument instead.

Reply:
Done.